composer.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "symfony/ux",
+    "license": "MIT",
+    "require": {
```

Comment:
should be `require-dev`, right?

Reply:
And `composer.lock` should not be committed

Reply:
Yeah it should be `required-dev`. And removing `composer.lock`